linter/spectral.yml

```diff
@@ -22,6 +22,9 @@ extends: spectral:oas
 rules:
   oas3-api-servers: error
 
+  #/core/no-trailing-slash
+  path-keys-no-trailing-slash: error
```

Comment:
Ik snap niet helemaal wat hier gebeurd is, maar ineens kwam deze error tevoorschijn in de laatste versie van Spectral. Dus onze eigen implementatie is hierdoor niet meer nodig.